salt/state.py

```diff
@@ -362,7 +362,6 @@ class StateError(Exception):
     '''
     Custom exception class.
     '''
-    pass
```

Comment:
Isn't this supposed to remain?

Reply:
As written in 332de26df886a442ccbfa43ccce1d73dd1b3bf96, this is to fix:
```
salt/state.py:370: [W0107(unnecessary-pass), StateError] Unnecessary pass statement
```

Ref:
- http://pylint.pycqa.org/en/latest/technical_reference/features.html
- https://help.semmle.com/wiki/display/PYTHON/Unnecessary+pass